python/composio/client/__init__.py

```diff
@@ -138,6 +139,21 @@ def http(self) -> HttpClient:
     @http.setter
     def http(self, value: HttpClient) -> None:
         self._http = value
+        
+    @property
+    def long_timeout_http(self) -> HttpClient:
```

Comment:
Consider adding a docstring to the `long_timeout_http` property to explain its purpose and the significance of the 180-second timeout. This will help future maintainers understand why this separate client exists and when to use it.